src/app/shared/components/header/header.component.html

```diff
@@ -14,15 +14,16 @@
         <ion-icon name="chevron-back-outline" slot="icon-only"></ion-icon>
       </ion-button>
     </ion-buttons>
-    <ng-container>
-      <ion-title
-        *ngIf="!(headerConfig().variant === 'compact')"
-        style="text-align: center"
-        routerLink="/"
-      >
+    @if (headerConfig().template) {
+      <plh-template-container
+        [templatename]="headerConfig().template"
+        [ignoreQueryParamChanges]="true"
+      ></plh-template-container>
+    } @else if (!(headerConfig().variant === "compact")) {
```

Comment:
nit(non-blocking): feels like a bit of a clunky if statement. So the first case is to show a template if the user includes a template, and the second case is to show the title text if the user both has not selected a template nor set the variant as compact.... I'm not sure if there is a way to tidy up the negation a bit, or better means to handle what to display.

I'm also slightly confused as to what is designed to be displayed if the author has specified a compact variant with a title?

Reply:
If the `compact` variant is specified then no title is displayed, even if the author has provided one.

I've improved this logic slightly in https://github.com/IDEMSInternational/open-app-builder/pull/2631/commits/d5fe6c086d2ab2eba22220b839662152b25d942f, which I think, makes it clearer